src/renderer/views/Watch/Watch.js

```diff
@@ -1518,6 +1526,25 @@ export default Vue.extend({
       return timestamp
     },
 
+    keyboardShortcutHandler: function (event) {
+      if (event.ctrlKey || document.activeElement.classList.contains('ft-input') || !event.shiftKey) {
+        return
+      }
+      switch (event.key.toUpperCase()) {
```

Comment:
Why we use `switch (event.key.toLowerCase()) {` above but `toUpperCase` here ?_?
No an issue for me though, just curious